src/main/kotlin/me/zeroeightsix/kami/target/CategorisedTargets.kt

```diff
@@ -119,6 +130,36 @@ enum class BlockCategory(override val belongsFunc: (Block) -> Boolean) : Categor
     override val provider: ResettableLazy<List<Block>?> = ResettableLazy { emptyList() }
 }
 
+@Suppress("unused")
+enum class ItemCategory(override val belongsFunc: (Item) -> Boolean) : CategorisedTargetProvider<Item> {
+    NONE({ false }),
+    TOOLS({ it is ToolItem }),
```

Comment:
Shears, fishing rod and flint and steel are items I would consider tools. They are not covered by `is ToolItem`